common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2169,6 +2170,16 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("5G")
 
+  val WORKER_DISK_RESERVE_RATIO: OptionalConfigEntry[Double] =
```

Comment:
As there is a config `celeborn.worker.storage.disk.reserve.size`. The meaning of "reserve" should be kept with the old one.